app/assets/javascripts/application.js

```diff
@@ -19,3 +19,7 @@
 $( document ).ready(function(){
   $(".button-collapse").sideNav();
 });
+
+$(document).ready(function() {
+    $('select').material_select();
```

Comment:
Mixed double and single quotes.<br>Identifier 'material_select' is not in camel case.